underfs/s3a/src/test/java/alluxio/underfs/s3a/S3AUnderFileSystemTest.java

```diff
@@ -239,4 +239,9 @@ public void getOperationMode() throws Exception {
     Assert.assertEquals(UfsMode.READ_WRITE,
         mS3UnderFileSystem.getOperationMode(physicalUfsState));
   }
+
+  @Test
+  public void stripPrefixIfPresent() throws Exception {
+    Assert.assertEquals("", mS3UnderFileSystem.stripPrefixIfPresent("s3a://" + BUCKET_NAME));
```

Comment:
What would the previous code have returned?

Reply:
The previous code would have returned s3a:// + BUCKET_NAME